blog-pipeline/tests/test_extracted_normalizers.py

```
@@ -176,6 +238,25 @@ def test_empty_keyword_falls_back_to_topic(self):
         assert result  # Should not be empty
         assert "hubspot" in result.lower()
 
+    def test_strips_bold_markers_before_normalization(self):
+        from blog_runner.normalizers import normalize_initial_keyword
+        result = normalize_initial_keyword("**revenue operations**", "fallback topic")
+        assert "**" not in result
+        assert "revenue operations" == result
```

Ruff (0.15.7)

[warning] Yoda condition detected

Rewrite as `result == "revenue operations"`

(SIM300)